modules/@apostrophecms/ui/ui/apos/components/AposSlatList.vue

```diff
@@ -110,8 +114,25 @@ export default {
         this.delayedDragging = false;
       });
     },
-    items() {
-      this.updateMessage();
+    value() {
+      this.next = this.value.slice();
+    },
+    next(newValue, oldValue) {
+      let equal = true;
+      if (newValue.length === this.value.length) {
```

Comment:
This logic prevents infinite loops while still allowing the parent to send a decorated version of its current list into the slat list. This makes it easy for the array editor to send in temporary fake titles like `#5`.

We only consider changes to the list length, the ids (order), and the title to be interesting because those are the legitimate concerns of the slat list.